packages/utils/src/feature_toggle.rs

```diff
@@ -0,0 +1,414 @@
+    }
+
+    fn _stop<S: Storage>(storage: &mut S, features: Vec<String>) -> StdResult<()> {
+        for f in features {
+            Self::_set_feature_status(storage, f, FeatureStatus::Stopped)?;
+        }
+
+        Ok(())
+    }
+
+    fn _resume<S: Storage>(storage: &mut S, features: Vec<String>) -> StdResult<()> {
+        for f in features {
+            Self::_set_feature_status(storage, f, FeatureStatus::Resumed)?;
+        }
+
+        Ok(())
+    }
+
+    fn _get_pauser<S: ReadonlyStorage>(storage: &S, key: HumanAddr) -> StdResult<Option<u8>> {
+        let feature_store =
+            ReadonlyBucket::multilevel(&[PREFIX_FEATURE_TOGGLE, PREFIX_PAUSERS], storage);
+        feature_store.may_load(key.0.as_bytes())
+    }
+
+    fn _set_pauser<S: Storage>(storage: &mut S, key: HumanAddr) -> StdResult<()> {
+        let mut feature_store =
+            Bucket::multilevel(&[PREFIX_FEATURE_TOGGLE, PREFIX_PAUSERS], storage);
+        feature_store.save(key.0.as_bytes(), &1_u8 /* value is insignificant */)
+    }
+
+    fn _remove_pauser<S: Storage>(storage: &mut S, key: HumanAddr) {
+        let mut feature_store: Bucket<S, u8> =
+            Bucket::multilevel(&[PREFIX_FEATURE_TOGGLE, PREFIX_PAUSERS], storage);
+        feature_store.remove(key.0.as_bytes())
+    }
+
+    fn _get_feature_status<S: ReadonlyStorage>(
+        storage: &S,
+        key: String,
+    ) -> StdResult<Option<FeatureStatus>> {
+        let feature_store =
+            ReadonlyBucket::multilevel(&[PREFIX_FEATURE_TOGGLE, PREFIX_FEATURES], storage);
+        feature_store.may_load(key.as_bytes())
+    }
+
+    fn _set_feature_status<S: Storage>(
+        storage: &mut S,
+        key: String,
+        item: FeatureStatus,
+    ) -> StdResult<()> {
+        let mut feature_store =
+            Bucket::multilevel(&[PREFIX_FEATURE_TOGGLE, PREFIX_FEATURES], storage);
+        feature_store.save(key.as_bytes(), &item)
+    }
+}
+
+#[derive(Serialize, Debug, Deserialize, Clone, JsonSchema, PartialEq)]
+pub enum FeatureStatus {
+    Resumed,
+    Stopped,
+}
+
+impl Default for FeatureStatus {
+    fn default() -> Self {
+        FeatureStatus::Resumed
+    }
+}
+
+#[derive(Serialize, Debug, Deserialize, Clone, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum FeatureToggleMsg {
+    Stop { features: Vec<String> },
+    Resume { features: Vec<String> },
+    SetPauser { address: HumanAddr },
+    RemovePauser { address: HumanAddr },
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+enum ResponseStatus {
+    Success,
+    Failure,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+enum HandleAnswer {
+    Stop { status: ResponseStatus },
+    Resume { status: ResponseStatus },
+    SetPauser { status: ResponseStatus },
+    RemovePauser { status: ResponseStatus },
+}
```

Comment:
We really don't need this pattern, in general. I know it's everywhere but, the `Failure` variant is never used. in general, I think that if we're not returning any data from the contract we can just return an empty object or something like that.